.circleci/config.yml

```diff
@@ -162,19 +162,81 @@ jobs:
                     rmdir "$extracted_folder"
                   fi
 
+      - run:
+          name: Check for index.html
+          command: |
+            cd ~/project/repository
+            OUTPUT_DIR="."
```

Comment:
not sure i understand why this logic seems so complex. is it not possible to have something like?

if OUTPUT_DIR != "" then
  cd OUTPUT_DIR
end

ls -la
if [ ! -f "index.html" ]; then
  ..
end

Reply:
It is to make the test work with the default values, only want to use the OUTPUT_DIR default value when the repo is the default one

Reply:
when you say test is that when a new commit is pushed to the runner repo it fails without  the default values?